ledger/src/blockstore_processor.rs

```diff
@@ -496,7 +507,7 @@ fn rebatch_and_execute_batches(
     let target_batch_count = get_thread_count() as u64;
 
     let mut tx_batches: Vec<TransactionBatchWithIndexes<SanitizedTransaction>> = vec![];
-    let rebatched_txs = if total_cost > target_batch_count.saturating_mul(minimal_tx_cost) {
```

Comment:
What's this change about?

Reply:
think this messed up some tests because it rebatches unexpectedly. I'll take a shot at preserving the previous behavior

Reply:
restored: 8eae1cce905c38a6bfa82fdf6fe9f21ed7654054